### PR TITLE
Coverage report, fixes #2

### LINKED ISSUE #2: Any luck with code coverage?
Since you're using a karma with browserify, I was wondering if you tried to add code coverage with `karma-coverage`. I've seen https://github.com/douglasduteil/isparta which uses a fork of `karma-coverage`, however when used with browserify, the line numbers between your source and your babelifyed code do not line up, throwing errors in test runner.

### PR DESCRIPTION
**Pull request by nkbt**
You coverage is pretty good!

I spent a good hour trying to make coverage working with browserify and then just replaced them with webpack and everything went super-smoooth. All fixed in literally 5 minutes. That's the best tooling so far!

screen shot 2015-06-18 at 23 19 56

### CODE REVIEW

**Comment by ajoslin** [comment]
I did `npm install` and `karma start`, but it's not giving me a coverage report, and I don't see a `coverage` folder anywhere. Is there a different command I need to run?

Also, I've had similar experiences with webpack. Once it's setup, it works damn well.

**Comment by nkbt** [comment]
I added npm script, so use: `npm run cov`

File: package.json [hunk 1]
```diff
@@ -2,19 +2,21 @@
   "name": "ng-classy",
   "version": "0.0.3",
   "scripts": {
-    "test": "./node_modules/.bin/karma start",
-    "test-watch": "./node_modules/.bin/karma start --single-run=false"
+    "test": "karma start",
+    "test-watch": "karma start --single-run=false",
+    "cov": "rm -rf ./coverage && node ./node_modules/karma/bin/karma start ./karma.conf.js --single-run --reporters coverage"
```
Comment (nkbt) [inline package.json]: `npm run cov` added

File: package.json [hunk 2]
```diff
@@ -2,19 +2,21 @@
   "name": "ng-classy",
   "version": "0.0.3",
   "scripts": {
-    "test": "./node_modules/.bin/karma start",
-    "test-watch": "./node_modules/.bin/karma start --single-run=false"
+    "test": "karma start",
```
Comment (nkbt) [inline package.json]: no need to prepend with `./node_modules/....` since npm does it automatically

**Comment by nkbt** [comment]
@ajoslin have you abandoned the project?

**Pull request closed without merging**